projects/plugins/crm/webpack.config.js

```diff
@@ -120,6 +120,10 @@ const crmWebpackConfig = {
 	},
 	resolve: {
 		...jetpackWebpackConfig.resolve,
+		alias: {
+			...jetpackWebpackConfig.resolve.alias,
+			"crm": path.resolve(__dirname, "src/js/")	
```

Comment:
It's unfortunate that crm is globally ignoring eslint. The suggestion it would make here seems useful.
```suggestion
			crm: path.resolve( __dirname, 'src/js/' ),
```